forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt

```diff
@@ -21,6 +24,10 @@ class ForagePINEditText @JvmOverloads constructor(
     defStyleAttr: Int = R.attr.foragePanEditTextStyle
 ) : ForageUI, LinearLayout(context, attrs, defStyleAttr) {
     private var vault: VaultWrapper
+    override var isValid: Boolean = false
+        get() = vault.manager.getState().isEmpty
```

Comment:
Same here.

Reply:
nice catch! Fixed in [396ff6a](https://github.com/teamforage/forage-android-sdk/pull/55/commits/396ff6aad67733f049b437435e51baa8385f0dee)